deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2017 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright 2018

Reply:
lol, making this change actually breaks the build due to the boilerplate python checker. Shouldn't they all change to 2018 in a separate commit?

Reply:
The date here refers to the date the copyright was first claimed for this code. Fixed via #93